_includes/scripts.html

```diff
@@ -12,13 +12,116 @@
 {% if site.ga_tracker %}
 
 <script>
-  window.ga = window.ga || function () { (ga.q = ga.q || []).push(arguments) }; ga.l = +new Date;
-  ga('create', '{{ site.ga_tracker }}', 'auto');
-  ga('require', 'eventTracker');
-  ga('require', 'maxScrollTracker');
-  ga('require', 'outboundFormTracker');
-  ga('require', 'outboundLinkTracker');
-  ga('send', 'pageview');
+  window.ga =
+    window.ga ||
+    function() {
+      (ga.q = ga.q || []).push(arguments);
+    };
+  ga.l = +new Date();
+
+  ga("create", "{{ site.ga_tracker }}", "auto");
+  ga("require", "eventTracker");
+  ga("require", "maxScrollTracker");
+  ga("require", "outboundFormTracker");
+  ga("require", "outboundLinkTracker");
+  ga("send", "pageview");
+
+  // Google Analytics Event Tracking for IAC Page A/B Tests
```

Comment:
This is a lot of JS code now. Perhaps put it into a `.js` file and load as an external script (rather than inline)?